FND/Agreements/Contracts.rdf

```diff
@@ -123,43 +125,39 @@
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-agr-ctr;ContractOriginator">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-agr-ctr;ContractPrincipal"/>
 		<rdfs:label>contract originator</rdfs:label>
-		<skos:definition>the party that originates the contract and acts as the principal in that contract regardless of the owner or counterparty</skos:definition>
+		<skos:definition>party that originates a contract and acts as principal</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-agr-ctr;ContractParty">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-pty-pty;PartyInRole"/>
 		<rdfs:label>contract party</rdfs:label>
-		<skos:definition>a party to the contract, that is a contractually capable person or organization which is a signatory to the contract, and which grants or concedes certain rights and obligations as defined in the contract</skos:definition>
+		<skos:definition>party to a contract, that is a legally competant person or organization that is a signatory to the contract, and that grants or concedes certain rights and obligations as defined in the contract</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-agr-ctr;ContractPrincipal">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-agr-ctr;ContractParty"/>
 		<rdfs:label>contract principal</rdfs:label>
-		<skos:definition>the party identified as being the principal or first party to a contract, in the event that the contract distinguishes any party as the principal</skos:definition>
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/principal.html</fibo-fnd-utl-av:definitionOrigin>
+		<skos:definition>party identified as being the principal or first party to a contract, in the event that the contract distinguishes any party as the principal</skos:definition>
 		<fibo-fnd-utl-av:explanatoryNote>In law, the principal is the party that has the primary responsibility in a liability or obligation, as opposed to an endorser, guarantor, or surety.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-agr-ctr;ContractThirdParty">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-pty-pty;PartyInRole"/>
 		<rdfs:label>contract third party</rdfs:label>
-		<skos:definition>someone who may be indirectly involved but is not a principal party to an arrangement, contract, deal, lawsuit, or transaction</skos:definition>
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/third-party.html</fibo-fnd-utl-av:definitionOrigin>
+		<skos:definition>someone that may be indirectly involved but is not a principal party to an arrangement, contract, deal, lawsuit, or transaction</skos:definition>
```

Comment:
This seems at odds with ContractPrincipal definition which implies there can be at most one Primary.